src/app/admin/admin-reports/filtered-items/filtered-items.component.ts

```diff
@@ -0,0 +1,301 @@
+                      fieldsRest.forEach(field => {
+                        let fieldName = schema.prefix + '.' + field.toString();
+                        let fieldVO = OptionVO.item(fieldName, fieldName);
+                        this.metadataFields.push(fieldVO);
+                        this.metadataFieldsWithAny.push(fieldVO);
+                        if (isEmpty(field.qualifier)) {
+                          fieldName = schema.prefix + '.' + field.element + '.*';
+                          fieldVO = OptionVO.item(fieldName, fieldName);
+                          this.metadataFieldsWithAny.push(fieldVO);
+                        }
+                      });
+                    }
+                  );
+            });
+          }
+        );
+  }
+
+  loadPredicates(): void {
+    this.predicates = [
+      OptionVO.item(QueryPredicate.EXISTS, 'admin.reports.items.predicate.exists'),
+      OptionVO.item(QueryPredicate.DOES_NOT_EXIST, 'admin.reports.items.predicate.doesNotExist'),
+      OptionVO.item(QueryPredicate.EQUALS, 'admin.reports.items.predicate.equals'),
+      OptionVO.item(QueryPredicate.DOES_NOT_EQUAL, 'admin.reports.items.predicate.doesNotEqual'),
+      OptionVO.item(QueryPredicate.LIKE, 'admin.reports.items.predicate.like'),
+      OptionVO.item(QueryPredicate.NOT_LIKE, 'admin.reports.items.predicate.notLike'),
+      OptionVO.item(QueryPredicate.CONTAINS, 'admin.reports.items.predicate.contains'),
+      OptionVO.item(QueryPredicate.DOES_NOT_CONTAIN, 'admin.reports.items.predicate.doesNotContain'),
+      OptionVO.item(QueryPredicate.MATCHES, 'admin.reports.items.predicate.matches'),
+      OptionVO.item(QueryPredicate.DOES_NOT_MATCH, 'admin.reports.items.predicate.doesNotMatch')
+    ];
+  }
+
+  loadPageLimits(): void {
+    this.pageLimits = [
+      OptionVO.item('10', '10'),
+      OptionVO.item('25', '25'),
+      OptionVO.item('50', '50'),
+      OptionVO.item('100', '100'),
+      OptionVO.item('250', '250'),
+      OptionVO.item('1000', '1000')
+    ];
+  }
+
+  queryPredicatesArray(): FormArray {
+    return (this.queryForm.get('queryPredicates') as FormArray);
+  }
+
+  addQueryPredicate(newItem: FormGroup = new QueryPredicate().toFormGroup(this.formBuilder)) {
+    this.queryPredicatesArray().push(newItem);
+  }
+
+  deleteQueryPredicateDisabled(): boolean {
+    return this.queryPredicatesArray().length < 2;
+  }
+
+  deleteQueryPredicate(index: number, nbToDelete: number = 1) {
+    if (index > -1) {
+      this.queryPredicatesArray().removeAt(index);
+    }
+  }
+
+  setPresetQuery() {
+    let queryField = this.queryForm.controls.presetQuery as FormControl;
+    let value = queryField.value;
+    let query = this.presetQueries.find(q => q.id === value);
+    if (query !== undefined) {
+      this.queryPredicatesArray().clear();
+      query.predicates
+          .map(qp => qp.toFormGroup(this.formBuilder))
+          .forEach(qp => this.addQueryPredicate(qp));
+      if (query.predicates.length == 0) {
```

Comment:
This should be a triple equals (`===`) instead of `==`. See https://howtodoinjava.com/typescript/equals-vs-strict-equals/